src/build.sh

```diff
@@ -130,7 +136,7 @@ function parse_build_options()
   options_values['ARCH']="${configurations[arch]:-'x86_64'}"
   options_values['MENU_CONFIG']=''
   options_values['CROSS_COMPILE']="${configurations[cross_compile]}"
-  options_values['PARALLEL_CORES']=1
+  options_values['OPTIMIZE']="${configurations[optimize]}"
```

Comment:
The associative array `configurations` is associated with `kworkflow.config` file, in this sense, you need to add `optimize` in the `etc/kworkflow_template.config`. Also, we can use this option to store user-specific flags.